### ISSUE TITLE
[x86/Linux] GC Stress Test with level 4

### ISSUE THREAD

**Issue by wateret** [dotnet/coreclr#11043]
GC Stress test with level less than 4 are fine but with level 4 looks like CLR requires CoreDisTools.

```
$ COMPlus_GCStress=4 /work/nfs/dotnet/t2/corerun /work/nfs/dotnet/unittest-p0/Windows_NT.x86.Release/GC/API/GC/RemoveMemoryPressureTest/RemoveMemoryPressureTest.exe
LoadLibrary failed for '/work/nfs/dotnet/t2/libcoredistools.so': error 126
External disassembler is not available.
coreclr_initialize failed - status: 0x80004005
```

**Comment by wateret** [comment]
On my x64/Linux machine, I was able to get `libcoredistools.so` with `tests/setup-runtime-dependencies.sh`. Is there a same file for x86/Linux?

In the script file, I've tried to change the `$rid` from ubuntu.14.04-x64 to ubuntu.14.04-x86 but I still get the x64 version.

**Comment by wateret** [comment]
I also tried to build LLILC but it is not easy to build it on x86. Is LLILC is the right source for CoreDisTools?

**Comment by janvorli** [comment]
Yes, it requires the libcoredistools.so - a disassembler - to figure out sizes of instructions so that it can insert an invalid instruction after each generated asm instruction to invoke a signal handler and trigger a GC. I think that llilc is the right source of the coredistools stuff.
But most of the GC related issues can be discovered with GC modes 1, 2 and 3, so if you want to use it now just as a tool for verifying the changes you are working on, then I'd recommend using just those three modes and leaving the support for the modes >=4 for later.

**Comment by RussKeldorph** [comment]
Yeah, we need to produce libcoredistools.so for Linux/x86.

@dotnet/jit-contrib @swaroop-sridhar FYI

**Comment by BruceForstall** [comment]
@zamont I was talking to you yesterday about how we need coredistools for more platforms; here's an issue related to Linux/x86, and @sergign60 has a need for this in the referenced issue.
